manage-data/migrate.md

```diff
@@ -107,86 +115,95 @@ Through the {{es}} reindex API, you can connect your new {{es}} Service deployme
 
 ::::{note}
 For {{ece}} users, while it is most common to have Amazon S3 buckets, you should be able to restore from any addressable external storage that has your {{es}} snapshots.
 ::::
 
-1. On your old {{es}} cluster, choose an option to get the name of your snapshot repository bucket:
+The following steps assume you already have a snapshot repository configured in the old cluster with at least one valid snapshot containing the data you want to migrate.
 
-    ```sh
-    GET /_snapshot
-    GET /_snapshot/_all
-    ```
+### Step 1: Set up the repository in the new cluster [migrate-repo-setup]
 
-2. Get the snapshot name:
+In this step, you’ll configure a snapshot repository in the new cluster that points to the storage location used by the old cluster. This allows the new cluster to access and restore snapshots created in the original environment.
 
-    ```sh
-    GET /_snapshot/NEW-REPOSITORY-NAME/_all
-    ```
+::::{tip}
+If your new {{ech}} or {{ece}} deployment cannot connect to the same repository used by your self-managed cluster, for example if it's a private NFS share, consider one of these alternatives:
 
-    The output for each entry provides a `"snapshot":` value which is the snapshot name.
+* [Back up your repository](/deploy-manage/tools/snapshot-and-restore/self-managed.md#snapshots-repository-backup) to a supported storage system such as AWS S3, Google Cloud Storage, or Azure Blob Storage, and then configure your new cluster to use that location for the data migration.
+* Expose the repository contents over `ftp`, `http`, or `https`, and use a [read-only URL repository](/deploy-manage/tools/snapshot-and-restore/read-only-url-repository.md) type in your new deployment to access the snapshots.
+::::
 
-    ```json
-    {
-      "snapshots": [
-        {
-          "snapshot": "scheduled-1527616008-instance-0000000004",
-          ...
-        },
-        ...
-      ]
-    }
+1. On your old {{es}} cluster, retrieve the snapshot repository configuration:
+
+    ```sh
+    GET /_snapshot/_all
     ```
 
+    Take note of the repository name and type (for example, `s3`, `gcs`, or `azure`), its base path, and any additional settings. Authentication credentials are often stored in the [secure settings](/deploy-manage/security/secure-settings.md) on each node. You’ll need to replicate all this configuration when registering the repository in the new ECH or ECE deployment.
+
+    If your old cluster has multiple repositories configured, identify the repository with the snapshots containing the data that you want to migrate.
 
-3. Add the snapshot repository:
+2. Add the snapshot repository on the new cluster:
 
-    ::::{tab-set}
+    The new cluster must register a snapshot repository that points to the same physical storage location used by the old cluster. This ensures the new cluster can access the existing snapshots.
```

Comment:
I think this is kind of a summary of the entire task (connecting the old snapshot repo to the new cluster) and therefore could be removed. let me know if this is serving a different purpose and I am misreading

Reply:
yes, it feels duplicate, thanks